src/components/Message.tsx

```diff
@@ -0,0 +1,43 @@
+import { Avatar, Box, Flex, Stack, Text, Title } from "@mantine/core";
+import dayjs from "dayjs";
+
+import styles from "./Message.module.css";
+
+export type MessageProps = {
+  message: {
+    id: string;
+    channel_id: string;
+    created_at: Date;
+    content: string;
+  };
+  user: {
+    id: string;
+    name: string;
+    icon_url: string | null;
+  };
+};
```

Comment:
```suggestion
import { Avatar, Box, Flex, Stack, Text, Title } from "@mantine/core";
import { Message as MessageModel, User } from "@prisma/client";
import dayjs from "dayjs";

import styles from "./Message.module.css";

export type MessageProps = {
  message: Pick<MessageModel, "id" | "channelId" | "createdAt" | "content">;
  user: Pick<User, "id" | "name" | "iconUrl">;
};
```
みたいな感じで書いた方が煩雑にならずに済むと思うのですがいかがでしょう…？

Reply:
たしかに！
(PrismaからうまくModelの型を引っ張って来れなかったので参考になります、ありがとうございます🙇‍♂️)